documentation/whatsnew/server/server-news-4.3.0.Alpha2.adoc

```diff
@@ -0,0 +1,16 @@
+= Server What's New in 4.3.0.Alpha2
+:page-layout: whatsnew
+:page-component_id: server
+:page-component_version: 4.3.0.Alpha2
+:page-feature_jbt_only: true
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Alpha2
+
+
+== Experimental support added for WildFly 9 full edition
```

Comment:
I've updated it for clarity. Now use the phrase "Wildfly 9 Application Server Distribution" and also add a note that support for "Wildfly 9 Servlet-only distribution" has not yet been added.